File: scripts/calcular_imc.go
package main

import (
	"fmt"
)

// Función para calcular el IMC, el peluca sapee cambio para generar confilcto
type Persona struct {
	Altura float64 // en metros
	Peso   float64 // en kilogramos
}

func (p Persona) CalcularIMC() float64 {
	return p.Peso / (p.Altura * p.Altura)
}

func main() {
	var altura, peso float64

	// Solicitar datos al usuario
	fmt.Print("Ingresa tu altura en metros: ")
	fmt.Scan(&altura)
	fmt.Print("Ingresa tu peso en kilogramos: ")
	fmt.Scan(&peso)

	persona := Persona{Altura: altura, Peso: peso}
	imc := persona.CalcularIMC()

	fmt.Printf("Tu IMC es: %.2f\n", imc)
}

File: scripts/calcular_imc.c
#include <stdio.h>

// Función para calcular el IMC, ALGO PARA VER SI GENERA CONFLICTOS
float calcularIMC(float peso, float altura) {
	return peso / (altura * altura);
}

int main() {
	float peso, altura;

	// Solicitar datos al usuario
	printf("Ingresa tu peso en kilogramos: ");
	scanf("%f", &peso);
	printf("Ingresa tu altura en metros: ");
	scanf("%f", &altura);

	float imc = calcularIMC(peso, altura);
	printf("Tu IMC es: %.2f\n", imc);

	return 0;
}